### ISSUE TITLE
TracyOpenGL.hpp GL header requirement

### ISSUE THREAD

**Issue by tksuoran** [wolfpld/tracy#228]
In TracyOpenGL.hpp
```
#if !defined GL_TIMESTAMP && !defined GL_TIMESTAMP_EXT
#  error "You must include OpenGL 3.2 headers before including TracyOpenGL.hpp"
#endif
```
Could you make it so that the GL headers are not required if `TRACY_ENABLE` is not defined? Thanks!

**Comment by wolfpld** [comment]
Fixed in e606c756.

**Comment by tksuoran** [comment]
Thanks!
